debian/rules

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/make -f
+
+export DH_VERBOSE=1
+
+export DH_OPTIONS
+
+export DH_GOPKG := github.com/awslabs/amazon-ecr-credential-helper
+
+VERSION := $(shell cat VERSION)
+GITCOMMIT_SHA := $(shell git rev-parse --short HEAD)
```

Comment:
If you really want to do this, you'll need to take extra steps to prevent `dpkg-source` from pruning your .git directory. As it is, if you try to run a build based on the debianized sources generated by the `docker-deb` script, you'll see:

```
debian/rules binary
fatal: not a git repository (or any of the parent directories): .git
```
(Note that it's not  actually fatal to the build, but GITCOMMIT_SHA is set to the empty string.)

See the `--diff-ignore` and `--extend-diff-ignore` flags to [dpkg-source](https://manpages.debian.org/stretch/dpkg-dev/dpkg-source.1.en.html) and note that `dpkg-buildpackage` will invoke it as `dpkg-source -i -I -b .` by default.

And if you do this, don't forget to add a lintian override to suppress the warning that it'd otherwise generate.

Reply:
Thanks, I thought I had changed this.  There should be a `GITCOMMIT_SHA` file in the orig tarball, so I should be able to do `GITCOMMIT_SHA := $(shell cat GITCOMMIT_SHA)`.  I will ensure that works and update.

Reply:
Fixed.